src/main/scala/edu/ie3/simona/service/em/EmServiceBaseCore.scala

```diff
@@ -56,83 +58,70 @@ import scala.jdk.CollectionConverters.{
   *   Option for em set points that needs to be handled at a later time.
   */
 final case class EmServiceBaseCore(
-    override val lastFinishedTick: Long = PRE_INIT_TICK,
+    override val lastFinishedTick: Long = INIT_SIM_TICK,
```

Comment:
Ergibt das Sinn? Normalerweise ist -1 ja der erste Tick, und der ist dann noch nicht finished

Reply:
Ok, but you get my point? A value called `lastFinishedTick` tells the developer that the tick saved has been finished before. But I think this doesn't add much to the overall complexity, which is already considerable.

Reply:
After some testing, I found that this is no longer needed. Therefore, I will remove this part.